### PR TITLE
GUVNOR-3557: [Guided Decision Table] User is able to bind two fields to same name

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-3557

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/ValueOptionsPageView.java [hunk 1]
```diff
@@ -95,6 +99,7 @@ public ValueOptionsPageView(final Div valueListGroupContainer,
         this.defaultValueContainer = defaultValueContainer;
         this.limitedValueContainer = limitedValueContainer;
         this.bindingContainer = bindingContainer;
+        this.fieldBindingWarning=fieldBindingWarning;
```
Comment (karreiro) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/pages/ValueOptionsPageView.java]: Here ;-)

**Comment by manstis** [comment]
@karreiro Updated. Thanks for your review :-)

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by manstis** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
screenshot from 2017-10-16 13-54-34

**Comment by manstis** [comment]
@jomarko Fixed. Sorry for wasting your time.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this.

**Comment by jomarko** [comment]
Also this, building locally, as jenkins ...

**Comment by manstis** [comment]
@jomarko As commented on the JIRA the reported issue is not related to this PR; however appears to be long-standing. I have however included a fix in this PR to remove bound ```FactPattern``` in "BRL Conditions" from the pattern selector drop-down when adding new "simple conditions". The enhancement to allow the addition of "simple conditions" to "BRL Conditions" is quite a bit more extensive and should probably be a new JIRA requesting such an enhancement, if desirable.

**Comment by jomarko** [comment]
@manstis sorry, I am reading this later, as jira, well I am fine if no new jira will be fired, if you mention the behavior here https://github.com/kiegroup/drools-wb/blob/master/drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties#L123

**Comment by manstis** [comment]
@jomarko I've updated PR to show different descriptions for "Pattern Page" depending on whether you're adding a condition (in which case only patterns on existing _Condition_ columns can be used); or whether you're adding an action (in which case the available patterns include those in both _Condition_ columns or _Condition BRL fragments_).

**Pull request merged**